### PR TITLE
Frontend Framework Restructuring

### PR DESCRIPTION
**Pull request by Reltre**
- removed bootstrap
- moved css to fies
- reproduced navContainer and Stats Container with Foundation

### CODE REVIEW

File: src/NavContainer.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
-import { Grid, GridItem } from 'styled-grid-component';
-import { Button } from 'react-bootstrap';
-
+import './NavContainer.css';
+import { Button, ButtonGroup } from 'react-foundation';
```
Comment (ian-r-rose) [inline src/NavContainer.js]: `react-foundation` needs to be added to the `package.json`.

**Comment by Reltre** [comment]
@ian-r-rose I can update those later today. @acostajake If you happen to get to that first, keep in mind that you'll have to push to my branch, that way changes will be reflected in this PR in upstream.

**Comment by Reltre** [comment]
@ian-r-rose Ok, this should be good to merge. I'm still thinking maybe we should add `package-lock.json` to version control, but we can decide on that later.

**Comment by acostajake** [comment]
Sorry to hop on this convo late in the day. Thanks again @Reltre and @ian-r-rose

**Comment by ian-r-rose** [comment]
We already have a `yarn.lock`, i don't think we should have more than one lockfile. Happy to switch to `package-lock.json`, however. What do you think @Reltre?

**Comment by ian-r-rose** [comment]
Thanks @Reltre and @acostajake!

**Pull request merged**